GetFed/GetFed/Controllers/FoodSearchViewController.swift

```diff
@@ -34,6 +41,34 @@ class FoodSearchViewController: UIViewController {
     }
 }
 
+// MARK - UITableViewDataSource & UITableViewDelegate Protocol Implementation
+
+extension FoodSearchViewController: UITableViewDataSource, UITableViewDelegate {
+    
+    func registerFoodResultTableViewCell() {
+        let cell = UINib(nibName: Identity.FoodResultTableViewCell.nibID, bundle: nil)
+        foodTableView.register(cell, forCellReuseIdentifier: Identity.foodSearchResultCell.cellID)
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        guard let rowCount = searchResults?.results.count else { return 0 }
+        if rowCount == 0 {
```

Comment:
You can actually use ternary expression here:
`return (rowCount == 0) ? 1 : rowCount`
It's a way of collapsing simple `if` expressions like this. It can be harder to read, so it should really only be used for a single expression like this.

Reply:
Agreed! Especially since this expression is so simple, as you pointed out.